### PR TITLE
fix(FEC-9167): set default IMA lib protocol for non http protocols

### PR DESCRIPTION
**Pull request by Yuvalke**
### Description of the Changes

Add default protocol https when our bundled application loaded with non valid protocol(not http or https).
Found on Samsung TV.

### CheckLists

- [ ] changes have been done against master branch, and PR does not conflict
- [ ] new unit / functional tests have been added (whenever applicable)
- [ ] test are passing in local environment 
- [ ] Travis tests are passing (or test results are not worse than on master branch :))
- [ ] Docs have been updated

### CODE REVIEW

File: src/ima.js [hunk 1]
```diff
@@ -584,9 +584,11 @@ class Ima extends BasePlugin implements IMiddlewareProvider, IAdsControllerProvi
    * @memberof Ima
    */
   _loadImaSDKLib(): Promise<*> {
+    const protocolRegex = /^(https:|http:)/i.exec(document.location.protocol);
+    const protocol = protocolRegex && protocolRegex.hasOwnProperty('index') ? protocolRegex[protocolRegex.index] : 'https:';
```
Comment (OrenMe) [inline src/ima.js]:
```suggestion
    const protocol = /^(https?:)/i.test(document.location.protocol) || "https:"
```

Comment (Yuvalke) [reply src/ima.js]: It's not good.
test method return true or false, so your protocol will return true instead the relevant protocol

Comment (OrenMe) [reply src/ima.js]: correct, it should have been
```suggestion
    /^(https?:)/i.test(document.location.protocol) ? document.location.protocol : "https:"
```

Comment (OrenMe) [reply src/ima.js]: and also add the `+':'` for `document.location.protocol`

**Pull request merged**